great_expectations/datasource/fluent/schemas/SnowflakeDatasource.json

```diff
@@ -51,8 +51,9 @@
                 },
                 {
                     "type": "string",
-                    "writeOnly": true,
-                    "format": "password"
+                    "minLength": 1,
+                    "maxLength": 65536,
+                    "format": "uri"
```

Comment:
This appears to be a duplicate of the following item?

Reply:
One represents the `ConfigUri` type and the other a standard `SnowflakeDsn` connection string.
I'll update this in a followup.
Either to distinguish these two items in the generated Schema, or remove the duplicate.

Reply:
@roblim 
https://github.com/great-expectations/great_expectations/pull/10023